docs/qiita-16S-analysis.rst

```diff
@@ -377,7 +377,11 @@ To filter the data, select the rarefied "Rarefied table 11030 (BIOM)" artifact i
    :align:   center
 
 Several parameters have been automatically selected for you. In the "SQLite WHERE-clause" field we are filtering out all samples except for certain samples. In this case we wanted to filter out all samples except those in which :code:`subject = 'Volunteer 3'`, and click "Add Command".
-If instead you want to filter out all of Volunteer 3's samples, insert :code:`subject != 'Volunteer 3'`, and click "Add Command". **Keep in mind that all fields are case sensitive**.
+If instead you want to filter out all of Volunteer 3's samples, insert :code:`subject != 'Volunteer 3'`, and click "Add Command". If you want to filter for samples containing an apostrophe, write it out in the following format:
+```
```

Comment:
should read instead:
::
     sample = \"Volunteer 3's samples\"

Reply:
see http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks for detailed formatting instructions

Reply:
Fixed the first and added the 2nd.